scripts/jobs/cron_tasks.inc.http.10.apache.php

```diff
@@ -838,12 +843,18 @@ protected function getVhostContent($domain, $ssl_vhost = false)
 		$query = "SELECT * FROM `" . TABLE_PANEL_IPSANDPORTS . "` `i`, `" . TABLE_DOMAINTOIP . "` `dip`
 			WHERE dip.id_domain = :domainid AND i.id = dip.id_ipandports ";
 		
+		// Ignore IPs which are proxied by NRP
```

Comment:
This comment would be better placed at line 170 and the following commented code removed (as it is exactly what is used above)